src/kaleidoscope/hid.cpp

```diff
@@ -79,22 +105,23 @@ void pressKey(Key mappedKey) {
   pressRawKey(mappedKey);
 }
 
-__attribute__((weak))
 void releaseRawKey(Key mappedKey) {
-  if (BootKeyboard.getProtocol() == HID_BOOT_PROTOCOL)
+  WITH_BOOTKEYBOARD_PROTOCOL {
     BootKeyboard.release(mappedKey.keyCode);
+  }
```

Comment:
Do we want to return here?

Reply:
We... kinda do. There's no problem if there isn't, because we'll just set the bits in the NKRO report, but never send it. But it is clearer and more correct if we do return. Pushing a patch that does that.

Thanks for catching it!